main.js

```diff
@@ -0,0 +1,140 @@
+let player = {
+  player1: {
+    name: "player1",
+    letter: "X",
+    winVal: 1 //when the abs(sum) of a row/col or diagonal is 3 the win condition is triggered
+  },
+  player2: {
+    name: "player2",
+    letter: "O",
+    winVal: -1
+  },
+  curentPlayer: "player1",
+  nextPlayer: "player2"
+};
+
+let start = document.getElementById("start1Player");
+let AIstart = document.getElementById("AIstart");
+let status = document.getElementById("turnIdentifier");
```

Comment:
I would like to see this as `turnIdentifier` but that's purely personal preference.  I like to make my variable names the same as the ID of the element I'm getting because it makes it easier for me to remember what I'm referencing